isis/src/base/apps/caminfo/caminfo.cpp

```diff
@@ -290,7 +290,7 @@ namespace Isis{
         // Run camstats on the entire image (all bands)
         // another camstats will be run for each band and output
         // for each band.
-        else if (ui.GetBoolean("CAMSTATS") && !incube->hasTable("CameraStatistics")) {
+        else if (ui.GetBoolean("CAMSTATS")) {
```

Comment:
I think this change breaks the functionality added in #4597. This will re-run the cam stats no matter what if `CAMSTATS=true`. What should be happening is that when CAMSTATS is on, it will use the CameraStatistics table if it exists and USECAMSTATSTBL is on. Otherwise, it will re-compute the statistics.

It's probably better to just wrap everything from line 250 to 345 in a `if (ui.GetBoolean("CAMSTATS"))` block. Then, the if at 251 can just be `if(ui.GetBoolean("USECAMSTATSTBL") &&  incube->hasTable("CameraStatistics"))` and this can just be an else.

Reply:
@jessemapel That would be true if this was in a separate `if` block on its own but it resides in an `else if` block. That  block only has the potential to run if the initial `if` statement and all other `else if` statements before it are not considered true.

I agree that adjusting the logic to better encapsulate what is happening in this section though. This was just the quickest way to handle this.

Reply:
Oops, my brain completely missed that! I think this is totally fine then. No need to re-arrange this.